### PR TITLE
extended example, added > to indicate ready for command input

### PR DESCRIPTION
**Pull request by jpihl**
@janusheide 
do you like it?

### CODE REVIEW

File: src/skipper/program.hpp [hunk 1]
```diff
@@ -106,7 +106,7 @@ namespace skipper
 
             std::ostringstream help;
             help << description << ", " << validate;
-            m_commands[key] = std::make_tuple(set_command, help.str());
+            add_command(key, description, set_command);
```
Comment (janusheide) [inline src/skipper/program.hpp:109]: I guess this is a mistake right?

**Comment by janusheide** [comment]
I think this is useful, but why dont we add it to program.hpp instead of only the example?

File: src/skipper/program.hpp (hunk 1 above)
Comment (jpihl) [inline src/skipper/program.hpp:109]: Um, no? :) I'm just reusing the other implementation..

**Comment by jpihl** [comment]
You wrote a comment directly on the PR, and not in the code, but i'm guessing you are referring to the `tuple_convert` lambda.
I agree with you that it would be useful to have in the `program.hpp` (or at least in the library somewhere) but to me there was no obvious way of making this a templated function so I didn't want to spend too much time on it. I just left it in the example to serve as exactly that; an example. We can always go back and look for a generic solution later when time is less of an issue.

**Pull request merged**